secator/hooks/gcs.py

```
@@ -45,7 +46,11 @@ def process_item(self, item):
 			t = Thread(target=upload_blob, args=(GCS_BUCKET_NAME, v, blob_name))
 			t.start()
 			self.threads.append(t)
-			setattr(item, k, f'gs://{GCS_BUCKET_NAME}/{blob_name}')
+			gcs_path = f'gs://{GCS_BUCKET_NAME}/{blob_name}'
+			setattr(item, k, gcs_path)
+			# Update type field for File objects
+			if item._type == 'file':
+				setattr(item, 'type', 'gcs')
```

Ruff (0.15.9)

[warning] Do not call `setattr` with a constant attribute value. It is not any safer than normal property access.

Replace `setattr` with assignment

(B010)